packages/shared/src/components/squads/SquadHeaderBar.tsx

```diff
@@ -27,53 +28,47 @@ export function SquadHeaderBar({
   className,
   ...props
 }: SquadHeaderBarProps): ReactElement {
-  const { user } = useAuthContext();
+  const [, copying, copyLink] = useCopySquadInvitation({ squad });
```

Comment:
The first item is the invitation's actual link which is not displayed anywhere.

> Also this should have some origin correct?

Do you know what should be the origin? Will check the taxonomy now.

Reply:
Just a reminder we as engineers should never come up with random new taxonomy items it will deeply affect analytics. (So should always use what's available or when in doubt consult with Sab)